source/install/install-rhel-66-mysql.rst

```diff
@@ -0,0 +1,66 @@
+..  _install-rhel-66-mysql:
+
+Installing MySQL Database Server
+================================
+
+Install and set up the database for use by the Mattermost server. You can install either MySQL or PostgreSQL. To install PostgreSQL, see :ref:`install-rhel-66-postgresql`.
```

Comment:
Clicking on this link brings me to this URL: https://jeffschering.github.io/mmdocs/monolith/install/install-ubuntu-1404-postgresql.html#install-ubuntu-1404-postgresql

As a result, I'm no longer on the install guide anymore.

Wondering if there's a way to link to the subsection on the existing install guide I'm on (essentially jump from MySQL to PostgreSQL guide on the same page)